src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -83,20 +83,63 @@ export function createLayerObject(parsedLayer, currentLayer) {
     }
     // format is : WMS|BASE_URL|LAYER_ID
     else if (parsedLayer.type === LayerTypes.WMS) {
-        const { year = null } = parsedLayer.customAttributes ?? {}
         // here we assume that is a regular WMS layer, upon parsing of the WMS get capabilities
         // the layer might be updated to an external group of layers if needed.
-        return new ExternalWMSLayer({
+        layer = new ExternalWMSLayer({
             id: parsedLayer.id,
             name: parsedLayer.id,
             opacity: parsedLayer.opacity ?? defaultOpacity,
             visible: parsedLayer.visible,
             baseUrl: parsedLayer.baseUrl,
             currentYear: year,
         })
+    } else {
+        // Finally check if this is a Geoadmin layer
+        layer = store.getters.getLayerConfigById(parsedLayer.id)?.clone()
+        if (layer) {
+            layer.visible = parsedLayer.visible
+            if (parsedLayer.opacity !== undefined) {
+                layer.opacity = parsedLayer.opacity
+            }
+            if (year !== undefined && layer.timeConfig) {
+                layer.timeConfig.updateCurrentTimeEntry(layer.timeConfig.getTimeEntryForYear(year))
+            }
```

Comment:
That block should also be moved down outside the `else`, `@year` on WMTS layer will be ignored otherwise

Reply:
No it should not, external layers WMS and WMTS are handled above using the year in their get capabilities, the else cover all geoadmin layers WMS/WMTS/GeoJSON, ...